app/plugins/file-log.md

```diff
@@ -77,9 +77,21 @@ Every request will be logged separately in a JSON object separated by a new line
             "access-control-allow-origin": "*"
         }
     },
+    "tries": [
```

Comment:
Yep, misread this. Thought this had to do with the log plugin retrying, not the retries in the serialized data. My bad!